src/datepicker/bs-datepicker-inline.component.ts

```diff
@@ -23,8 +23,16 @@ export class BsDatepickerInlineDirective implements OnInit, OnDestroy, OnChanges
     if (this._bsValue === value) {
       return;
     }
-    this._bsValue = value;
-    this.bsValueChange.emit(value);
+
+    let date = value;
+
+    if (date) {
+      const timezoneOffset = value.getTimezoneOffset() * 60000;
```

Comment:
Is it possible to have service to configure custom timezone offset?

Reply:
Maybe as different feature-request